content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -66,6 +66,7 @@ Additional controllers include:
 
 * [Contour](https://github.com/heptio/contour) is an [Envoy](https://www.envoyproxy.io) based ingress controller
   provided and supported by Heptio.
+* [Gloo](https://gloo.solo.io) is an open-source ingress controller based on [Envoy](https://www.envoyproxy.io) which offers API Gateway functionality with enterprise support from [solo.io](https://www.solo.io)
 * F5 Networks provides [support and maintenance](https://support.f5.com/csp/article/K86859508)
```

Comment:
@christian-posta, are the entries listed alphabetically? To be consistent, you could add a period at the end.

Reply:
Good catch @kbhawkey I've pushed an update